packages/appstore/src/components/app.scss

```diff
@@ -1,3 +1,33 @@
 .dw-dashboard {
     height: 100%;
 }
+
+@keyframes triggerAnimation {
```

Comment:
Just a suggestion: This can be created by the mixin. I create an example for it [here](https://gist.github.com/hamid-deriv/e81dc1f8709041a219cd113ac4608e62).

Reply:
That's a really good suggestion but it makes the code hard to read TBH. I think having a more readable code is more important than reducing some lines of code and make it hard to read.